.github/actions/upload/examples/README.md

```diff
@@ -0,0 +1,204 @@
+# Workflow Examples
+
+This directory contains example workflows for using the Filecoin Upload Action in different scenarios.
+
+## 📂 Directory Structure
+
+```
+examples/
+├── README.md                     # This file
+├── single-workflow/              # Simple, same-repo PRs only
+│   └── upload.yml
+└── two-workflow-pattern/         # Secure workflow pattern
+    ├── build-pr.yml              # Untrusted build workflow
+    └── upload-to-filecoin.yml    # Trusted upload workflow
+```
+
+## 🚀 Quick Start
+
+### Recommended: Two-Workflow Pattern
+
+✅ **This is the recommended and default pattern.**
+
+**Use when:**
+- You want maximum security
+- You want the secure default behavior
+
+**Setup:**
+1. Copy both files from `two-workflow-pattern/` to `.github/workflows/` in your repo
+2. Set `WALLET_PRIVATE_KEY` secret in your repository settings
+3. Update the build steps in `build-pr.yml` to match your project
+4. Adjust hardcoded `minStorageDays` and `filecoinPayBalanceLimit` in `upload-to-filecoin.yml` to your needs
+
+**That's it!** The action automatically handles:
+- ✅ Building CAR files from your build artifacts
+- ✅ Uploading to Filecoin with proper security
+- ✅ Commenting on the PR with results
+
+**Security:** ✅ Financial parameters are hardcoded in the trusted workflow.
+
+**This is the only pattern shown in the main README.** The single-workflow pattern is available but not recommended.
```

Comment:
I feel like maybe there's a heading missing here for the single workflow? we jump from "trusted" to "no recommended" and a scary security warning below ("don't accept fork PRs." - most people with open repos do). But if this section had `### Single workflow` then it might make more sense?